msr_whitelist.c

```diff
@@ -65,7 +65,7 @@ static struct whitelist_entry *find_in_whitelist(u64 msr);
 static void add_to_whitelist(struct whitelist_entry *entry);
 static int parse_next_whitelist_entry(char *inbuf, char **nextinbuf, struct whitelist_entry *entry);
 static ssize_t read_whitelist(struct file *file, char __user *buf, size_t count, loff_t *ppos);
-static int majordev;
+static int majordev_msr_whitelist;
```

Comment:
See previous comment for msr_batch.c.  You should be able to simply pass this value in as a parameter to this file's init() and cleanup() functions.